src/Sylius/Component/Core/Model/OrderItem.php

```diff
@@ -57,4 +57,24 @@ public function equals(BaseOrderItemInterface $item)
             && $item->getVariant() === $this->variant
         );
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments on order item and total tax of units.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
```

Comment:
Yeah just looking at having to add these methods at all 3 levels suggests the design is wrong to me